integration/localstack/ls_tmp/snakeoil.conf

```diff
@@ -14,4 +14,5 @@ CN = localhost
 subjectAltName = @alt_names
 [alt_names]
 DNS.1 = localhost
-DNS.2 = localhost.localstack.cloud
\ No newline at end of file
+DNS.2 = localhost.localstack.cloud
+DNS.3 = *.us-west-2.compute.amazonaws.com
```

Comment:
Do we need to specify `us-west-2` for the DNS? Part of me thinks we could/should be able to point this to any other commercial region if necessary.

Reply:
It seems you can’t wildcard sub domains for ssl
So *.domain.com works but
*.*.domain will not work
https://cheapsslsecurity.com/blog/how-to-secure-multi-level-subdomains/#:~:text=What%20is%20a%20Wildcard%20Certificate,any%20website%20with%20multiple%20subdomains.